pyproject.toml

```diff
@@ -847,6 +847,9 @@ addopts = [
     "--ignore-glob=**/tests/system/*",
     "--ignore-glob=tests/system/*",
 ]
+markers = [
+    "timeout: mark tests with a timeout (provided by pytest-timeout)",
+]
```

Comment:
I'm surprised we needed to add this -- we have lots of tests with timeouts already... 🤔